src/operator/nn/mkldnn/mkldnn_act.cc

```diff
@@ -79,18 +93,30 @@ mkldnn::algorithm GetMKLDNNActAlgo(const ActivationParam& param) {
   }
 }
 
+mkldnn::algorithm GetMKLDNNActAlgo(const LeakyReLUParam& param) {
+  switch (param.act_type) {
+    case leakyrelu::kLeakyReLU:
+      return mkldnn::algorithm::eltwise_relu;
+    case leakyrelu::kELU:
+      return mkldnn::algorithm::eltwise_elu;
+    default:
+      LOG(FATAL) << "unknown activation type";
```

Comment:
More descriptive error message"
```suggestion
      LOG(FATAL) << "unknown activation type for LeakyReLU: " << param.act_type;
```